src/components/resource/edit-custom-title/resource-edit-custom-title.js

```diff
@@ -244,6 +251,15 @@ class ResourceEditCustomTitle extends Component {
                   {resourceSelected ? (
                     <Fragment>
                       <VisibilityField disabled={visibilityMessage} />
+                      <div>
+                        {(!hasInheritedProxy || !proxyTypes.request.isResolved) ? (
```

Comment:
here, if a resource does not have inherited proxy, then we probably do not want to keep spinning. I am not sure ..is it possible that a resource can have proxy but the package containing it might not have one? If its possible, then we have to modify this check, otherwise, I think we can leave it as is.